udata_front/theme/gouvfr/datagouv-components/src/components/AppLink/AppLink.vue

```diff
@@ -0,0 +1,24 @@
+<template>
+  <a v-if="isExternalLink(to)" v-bind="$attrs" :href="to" target="_blank">
```

Comment:
I think we must change this `target` otherwise a dataset card opens all its links in a new tabs. This is not the intended behavior when there is no router.

When there is no router, I think we can leave the target handling to the caller.

I think, the easiest way to do it is to make a variable based on [useAttrs()](https://vuejs.org/guide/components/attrs#accessing-fallthrough-attributes-in-javascript) and add the target when we don't have a router